android/KMEA/app/src/main/assets/keyboard.html

```diff
@@ -16,13 +16,13 @@
   -->
   <script src="es6-shim.min.js"></script>
   <script src="keymanandroid.js"></script>
+  <script src="sentry.min.js"></script>
   <script src="keyman-sentry.js"></script>
```

Comment:
Are we needing to include both `sentry.min.js` and `keyman-sentry.js`?

I thought `keyman-sentry.js` was just the renamed component

Reply:
> - In converting our Sentry-integration setup to modular form... well... turns out we can't quite directly `import` the Sentry libraries and directly integrate with them.  
>    - `esbuild` falls over due to unsupported code transformation rules when targeting ES5, which we do b/c of our Android 5.0 support (and possibly other environments, too)
>    - We need to import `KEYMAN_VERSION` stuff, so... we'll either have to prepend bundled Sentry or include it in the asset bundles directly.
>    - I opted for the latter and have made the necessary changes for both mobile apps.

Reply:
Yes, this is the appropriate answer -- there's no need to concatenate these files since they are used only within Keyman Engine for Android and Keyman Engine for iOS.